src/components/AddItems.js

```diff
@@ -91,8 +92,15 @@ const AddItemsToList = () => {
   return (
     <div>
       <div>
+        <header className="bg-green-400 w-full fixed text-center">
+          <h2 className="pt-6 pb-16 text-4xl font-thin text-gray-100">
+            Add Item to List
+          </h2>
+          <span className="text-white top-0 right-0 absolute sm:mt-4 sm:mr-4">
+            <HomeIcon />
+          </span>
+        </header>
```

Comment:
Nice job with the header. A few observations. The header on the Additems component is a little different from the Iteslist header as seen in the screenshot. Also the texts arent centered well.
![Screenshot (119)](https://user-images.githubusercontent.com/57858821/108599264-02180b80-7345-11eb-8ce2-9cb8b239ff09.png)
![Screenshot (120)](https://user-images.githubusercontent.com/57858821/108599265-03e1cf00-7345-11eb-9874-99b5be9c523b.png)

Reply:
First, where has the form even gone? 😆 What did I do? I will check that out immediately!!
Do you mean the Heading isn't centered? When I inspect with dev tools, it seems to be centered.
Speaking of difference, do you mean bc of the bottom of the heading? This effect should come from the body your team created, that's why it is not shown yet.

Reply:
Also for the **color key**, I saw a plugin for styling the radio buttons, so we could use color on them.

Reply:
I love the button style with the little trashcan in it :slightly_smiling_face:

Reply:
I mean the height of the two headers(the green part). they are not equal . I think they should be equal in the both components irrespective of other contents of the components🙂

Reply:
I see. On my side, they seem the same size from the beginning :) I added the effect which should come from your code and now it looks equal as expected. (The body lays over the header - the plain header is bigger than shown when the body bc it is underneath the body)